### PR TITLE
Introduce fuzziness  for redacted timestamps

### PR DESCRIPTION
**Pull request by cburkert**
Concept described in #19

### CODE REVIEW

File: app/scripts/fuzzydate.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      }
+    case Timeunit.Hour:
+      if (time) {
+        format.hour = "2-digit"
+        format.timeZoneName = "short"
+      }
+    case Timeunit.Day:
+      format.day = "numeric"
+    case Timeunit.Month:
+      format.month = "short"
```
Comment (fapdash) [inline app/scripts/fuzzydate.js:27]: I think the longer month format would look better. WDYT?

**Comment by fapdash** [comment]
Using the back button breaks the redaction for me. :x

EDIT: It's only the redaction popup that breaks.
Example:

https://user-images.githubusercontent.com/459631/118722394-69523880-b82c-11eb-8c69-aad55a9053a2.mp4

File: app/scripts/fuzzydate.js (hunk 1 above)
Comment (cburkert) [reply app/scripts/fuzzydate.js:27]: GitHub itself uses the short month format. We'd run into overflow issues with the long format. I think the short format is fine.
Bildschirmfoto 2021-05-19 um 09 06 49

**Comment by cburkert** [comment]
> Using the back button breaks the redaction for me. :x

Should be fixed now. Please check @fapdash

**Pull request merged**